package_manager/cloudbuild.yaml

```diff
@@ -0,0 +1,30 @@
+# A cloud build config to release a PAR binary of
+# the dpkg_parser.par used for the package manager bazel rules
+
+# Takes a substitution of _VERSION
```

Comment:
Does this substitution work via triggers? We can set this up to run automatically everytime a PR is merged.